functions/format_message_test.ts

```diff
@@ -0,0 +1,29 @@
+import { SlackFunctionTester } from "deno-slack-sdk/mod.ts";
+import { assertEquals, assertStringIncludes } from "testing/asserts.ts";
+import FormatMessage from "./format_message.ts";
+
+/**
+ * The actual outputs of a function can be compared to expected outputs for a
+ * collection of given inputs.
+ * Learn more: https://api.slack.com/automation/functions/custom#testing
+ */
+const { createContext } = SlackFunctionTester("format_message");
+
+Deno.test("Mention the kind kudo receiver", async () => {
+  const inputs = {
+    doer_of_good_deeds: "USLACKBOT",
```

Comment:
wait you can just call slackbot like this

Reply:
🙏 all praises to the kind @USLACKBOT - it's a great placeholder that means something meaningful! To me it means everything.